server/data/projects.js

```diff
@@ -74,11 +81,13 @@ const createProject = async (projectObjParam, user) => {
 		media,
 		deploymentLink,
 		createdAt: date,
+		updatedAt: date,
 		technologies,
 		owner: userInfo,
 		savedBy: [],
 		comments: [],
 		likes: [],
+		imageArr: [],
```

Comment:
there is already an array for this, called `media` which is being initialized above